tests/test_selection/conftest.py

```diff
@@ -55,3 +55,13 @@ def df_test_num_cat():
     X = df[["var_A", "var_B", "var_C", "var_D"]]
     y = df["target"]
     return X, y
+
+
+@pytest.fixture(scope="module", name="dummy_method")
+def wrap_dummy_method():
+    import random
+
+    def _dummy_method(x, y):
+        return random.uniform(0.5, 1)
+
```

Comment:
@ChristopherGS @TremaMiguel 

Sorry, I am a bit ignorant on this front. I guess we need to wrap the method so that we do not need to pass X, y when calling it from the tests, is this correct? otherwise, why do we need to wrap the method?

would random be imported at every single evaluation of _dummy_method() ? would it not be better to import it at the top?

And as per previous message, could we give this block of code a more representative name? Something like "return_uniform_between_05_and_1" or something on that front.

Reply:
> 
> 
> Hi @solegalli any update with the check ?

Hey sorry for the belated response. I am on hols, back in the office on Monday. 

I think this PR is almost ready to merge. I made a PR to your repo @tremaMiguel with some minor edits to docstrings, and I have a question regarding the dummy_method, that would be great if you or @ChristopherGS could enlighten me with and/or make the changes if necessary.

Thanks a lot!

Reply:
yes importing random at the top would be better in this case as several tests can call this method, thanks for the catchup.

Reply:
Basically, we're wrapping it to make it accessible for the test function or inject it to the test function. 

The output of `wrap_dummy_method` is a callable `_dummy_method` (we do not pass any arguments to it yet) that it is passed as argument to `SmartCorrelatedSelection` in `test_callable_method`. 

Inside `SmartCorrelatedSelection`, the function `_dummy_method` is called with the provided `X, y` arguments from the user, basically inside `pandas.corr` is performed this calculation.

Reply:
@solegalli added changes requested, let me know if anything else is needed? Do you know if it is possible to change the target branch to `master` ?